### PR TITLE
SC-88445: add-code-coverage-to-all-dx-projects

### PR DESCRIPTION
**Pull request by neo-anand-patel**
Ticket:
> https://app.shortcut.com/neofinancial/story/88445/add-code-coverage-to-all-dx-projects

Changes Made:
> Added code coverage by configuring GitHub workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,155 +1,156 @@
...
-
-## Credits
-
-This project was inspired by [config3](https://github.com/focusaurus/config3) and [config4](https://github.com/autolotto/config4).
+# Config Dug
+
+[![Build status](https://github.com/neofinancial/config-dug/workflows/CI/badge.svg)](https://github.com/neofinancial/config-dug/actions)
+[![codecov](https://codecov.io/gh/neofinancial/config-dug/branch/master/graph/badge.svg)](https://codecov.io/gh/neofinancial/config-dug)
+![TypeScript 3.7.2](https://img.shields.io/badge/TypeScript-3.7.2-brightgreen.svg)
+[![coverage](https://coverage.neotools.ca/api/badge/master/4C8hhW7NzS4PKdA9C3NqNM)](https://coverage.neotools.ca/coverage/neofinancial/config-dug)
```
Comment (neo-anand-patel) [inline README.md:6]: I'm not sure why it re-added the entire READ.ME file but this is the only line that I added.

**Review by leah-is-offline — APPROVED** [review]
looks good. I think that codecov bot will run until endpoint is changed?

File: README.md (hunk 1 above)
Comment (iansu) [reply README.md:6]: I would try reverting your changes to this file and the reapplying them. It's weird that it's showing the whole file as changed.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -24,8 +29,15 @@ jobs:
       - name: Build
         run: npm run build
       - name: Test
-        run: npm run test --coverage
+        run: npm run test -- --coverage
       - name: Upload coverage
```
Comment (iansu) [inline .github/workflows/ci.yml:33]: Change this name to `Codecov`

**Review by iansu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**